OloEngine/src/OloEngine/Physics3D/JoltScene.cpp

```
@@ -324,6 +328,86 @@ namespace OloEngine
         return Entity{};
     }
 
+    JPH::BodyID JoltScene::CreateTerrainBody(Entity entity, const JPH::Ref<JPH::Shape>& shape,
```

Cppcheck (2.21.0)

[style] The function 'CreateTerrainBody' is never used.

(unusedFunction)